clayui.com/src/utils/localStorage.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const isSupported = () => {
+	try {
+		window.localStorage.setItem('_TEST_KEY_', '_TEST_KEY_');
```

Comment:
No `removeItem`?

Reply:
Probably slipped through the cracks while simplifying. Gonna take a look in a moment!